CHANGELOG.md

```diff
@@ -2,7 +2,17 @@
 
 Full documentation for hipTensor is available at [rocm.docs.amd.com/projects/hiptensor](https://rocm.docs.amd.com/projects/hipTensor/en/latest/index.html).
 
-## (Unreleased) hipTensor 1.4.0 for ROCm 6.3.0
+## hipTensor 1.5.0 for ROCm 6.4.0
+
+### Additions
+
+* Added benchmarking suites for contraction, permutation, and reduction. YAML files are categorized into bench and validation folders for organization.
+
+### Changes
+
+* Optimized the hyper-parameter selection algorithm for permutation.
+
```

Comment:
Should add a summary list of bug fixes as well

Reply:
done